From 716aa806cdc64aa71ede35ab57c2608e45189a92 Mon Sep 17 00:00:00 2001
From: Ke Qinghua <qinghua.ke@freescale.com>
Date: Tue, 22 Apr 2014 17:27:32 +0800
Subject: [PATCH 308/602] ENGR00305254 imx6sx sarbesd board bring up

Fix Sabreauto board NAND flash layout mismatch with mfg tool.

Signed-off-by: Ke Qinghua <qinghua.ke@freescale.com>
---
 sabreauto_6q/BoardConfig.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index 9818ded..919f407 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -93,7 +93,7 @@ TARGET_MKUBIFS_ARGS := -m 4096 -e 516096 -c 4096 -x none
 TARGET_UBIRAW_ARGS := -m 4096 -p 512KiB $(UBI_ROOT_INI)
 
 # Note: this NAND partition table must align with MFGTool's config.
-BOARD_KERNEL_CMDLINE +=  mtdparts=gpmi-nand:16m(bootloader),16m(bootimg),128m(recovery),-(root) ubi.mtd=4
+BOARD_KERNEL_CMDLINE +=  mtdparts=gpmi-nand:16m(bootloader),16m(bootimg),16m(recovery),-(root) ubi.mtd=4
 endif
 
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
-- 
1.8.0

